core/src/main/java/com/adobe/aem/guides/spascreens/core/models/impl/ProductImpl.java

```diff
@@ -232,9 +235,27 @@ public String getImage() {
 		Resource featureRes = featuresList.get(0);
 		ProductFeature feature = new ProductFeatureImpl(featureRes.getValueMap());
 		String imageUrl = feature != null ? feature.getImagePath() : null;
-
 		return imageUrl;
 	}
+	
+	@Override
+	public List<ImageMaps> getImageMaps() {
+		List<ImageMaps> imageMaps = new ArrayList<>();
```

Comment:
We should move this to a private function (e.g. readHotspots) and call it in the initModel() method